src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -187,25 +187,6 @@ End Class
             End Using
         End Function
 
-        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
-        Public Async Function CompletionDismissedAfterEscape1() As Task
```

Comment:
💭 Where has this gone to?

Reply:
Covered by the editor in the new implementation.

---
In reply to: [208003803](https://github.com/dotnet/roslyn/pull/29016#discussion_r208003803) [](ancestors = 208003803)